demisto_sdk/commands/secrets/secrets.py

```diff
@@ -263,7 +263,7 @@ def get_related_yml_contents(self, file_path):
         # if script or readme file, search for yml in order to retrieve temp white list
         yml_file_contents = ''
         # Validate if it is integration documentation file or supported file extension
-        if checked_type(file_path, REQUIRED_YML_FILE_TYPES):
+        if find_type(file_path) in [FileType.PYTHON_FILE, FileType.README]:
```

Comment:
we might need to add powershell here as well as it has a related yml

Reply:
I believe this command was written simply before we had ps1 support but looking at the code it seems legit that ps1 related yml files will be checked here - can we test what will happen?